src/ol_infrastructure/applications/concourse/Pulumi.applications.concourse.operations.CI.yaml

```diff
@@ -0,0 +1,25 @@
+secretsprovider: awskms://alias/infrastructure-secrets-qa
+encryptedkey: AQICAHjEYf3gV5ZTnviCJcHwGnUuLzld5EuYumgMrRde6bvXUgE9rvKwMh36kigoyf0oNRRpAAAAfjB8BgkqhkiG9w0BBwagbzBtAgEAMGgGCSqGSIb3DQEHATAeBglghkgBZQMEAS4wEQQM3ptgm6NCCpX9CmtCAgEQgDseumxlwIAYWyEvpYheyiv6TaJwKSlDTcyCRfrJ30eGefNBoi3DJK6NfWf2V7R6RsCBq8uoJfoOL/Cshw==
+config:
+  aws:region: us-east-1
+  concourse:consul_credentials:
+    secure: v1:5KxQDZzjcrkKQUro:V/Yfx056dylqBL25WIutQw1i3DCc18CMUCyyI/7DfwV7ZNkBlh6qXW8/UAUxIHVPxs9zlkJXTp6u51GNO4Td
+	concourse:db_instance_size: db.m6g.large
```

Comment:
We can leave this configuration unset for the CI environment, so this line can be deleted.